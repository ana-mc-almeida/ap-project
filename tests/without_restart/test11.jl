include("./common.jl")

handling(Error1 => (e) -> println("Error1"), Error2 => (e) -> println("Error2")) do
    handling(Error3 => (e) -> println("Error3"), Error4 => (e) -> println("Error4")) do
        raise_error(1)
    end
end

