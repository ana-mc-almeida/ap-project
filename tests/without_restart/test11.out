Error1
ERROR: Error1()